### PR TITLE
Updated Hub release notes

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updates to Hub Release notes

### CODE REVIEW

File: docker-hub/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,37 @@ known issues for each Docker Hub release.
...
+
+### New feature
+
+Registry Access Management is now available for all Docker Business subscriptions. When enabled, your users can access to specific registries in Docker Hub.
+
+## 2022-05-03
+
+### New feature
+
+Organization owners can invite new members to an organization via Docker ID, email address.
```
Comment (craig-osterhout) [inline docker-hub/release-notes.md:42]:
```suggestion
Organization owners can invite new members to an organization via Docker ID or email address.
```

File: docker-hub/release-notes.md [hunk 2]
```diff
@@ -11,6 +11,37 @@ known issues for each Docker Hub release.
...
+## 2022-7-22
+### Enhancement
+
+You can now invite members to your organization with a CSV file containing their email addresses. The CSV file can either be a file you create for this specific purpose or one that’s extracted from another in-house system.
+
+## 2022-07-19
+
+### Enhancement
+
+SCIM is now available for organizations with a Docker Business subscription using an Azure Ad identity provider.
```
Comment (craig-osterhout) [inline docker-hub/release-notes.md:24]:
```suggestion
SCIM is now available for organizations with a Docker Business subscription using an Azure AD identity provider.
```

File: docker-hub/release-notes.md [hunk 3]
```diff
@@ -11,6 +11,37 @@ known issues for each Docker Hub release.
...
+
+### New feature
+
+With SCIM, you can manage users within your Okta identity provider (IdP). In addition, you can enable SCIM on organizations that are part of the Docker Business subscription.
+
+## 2022-05-24
+
+### New feature
+
+Registry Access Management is now available for all Docker Business subscriptions. When enabled, your users can access to specific registries in Docker Hub.
```
Comment (craig-osterhout) [inline docker-hub/release-notes.md:36]:
```suggestion
Registry Access Management is now available for all Docker Business subscriptions. When enabled, your users can access specific registries in Docker Hub.
```

File: docker-hub/release-notes.md [hunk 4]
```diff
@@ -11,6 +11,37 @@ known issues for each Docker Hub release.
...
+
+### Enhancement
+
+SCIM is now available for organizations with a Docker Business subscription using an Azure Ad identity provider.
+
+## 2022-06-23
+
+### New feature
+
+With SCIM, you can manage users within your Okta identity provider (IdP). In addition, you can enable SCIM on organizations that are part of the Docker Business subscription.
```
Comment (usha-mandya) [inline docker-hub/release-notes.md:30]: Can we also add a link to the SCIM topic from the release notes? Suggest we do the same for other features in this PR so users can find more information if required.

File: docker-hub/release-notes.md (hunk 1 above)
Comment (jerae-duffin) [reply docker-hub/release-notes.md:42]: Thanks Craig! Updated

File: docker-hub/release-notes.md (hunk 2 above)
Comment (jerae-duffin) [reply docker-hub/release-notes.md:24]: Updated

File: docker-hub/release-notes.md (hunk 3 above)
Comment (jerae-duffin) [reply docker-hub/release-notes.md:36]: Updated

File: docker-hub/release-notes.md (hunk 4 above)
Comment (jerae-duffin) [reply docker-hub/release-notes.md:30]: Thanks Usha! Updated

File: docker-hub/release-notes.md [hunk 5]
```diff
@@ -11,6 +11,36 @@ known issues for each Docker Hub release.
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+
+## 2022-7-22
```
Comment (craig-osterhout) [inline docker-hub/release-notes.md:15]:
```suggestion
## 2022-07-22
```

**Review by craig-osterhout — DISMISSED** [review]
LGTM. 
Added a small nitpick (date format) that I didn't even notice the first time. I don't think it's a blocker.

**Review by craig-osterhout — APPROVED** [review]

**Pull request merged**